test/JDBC/expected/BABEL-5119-vu-prepare.out

```diff
@@ -0,0 +1,29 @@
+-- tsql
+CREATE DATABASE BABEL5119_1
+GO
+
+USE BABEL5119_1
+GO
+CREATE TABLE BABEL5119_t1(a int)
+GO
+CREATE TABLE BABEL5119_t2(a int)
+GO
+CREATE TABLE BABEL5119_t3(a int)
+GO
+INSERT INTO BABEL5119_t1 VALUES(1)
```

Comment:
Insertion should be performed in verify file not create until we don't use the inserted value in this script itself.

Reply:
Removed insert query from the vu-prepare file